.travis.yml

```diff
@@ -2,7 +2,7 @@ language: go
 
 go:
   - 1.6
-  - tip
+  - 1.7
```

Comment:
I think tip is going to be bad for a while, and we don't care about tip much yet.